### PR TITLE
Make bealign parallelism configurable

### PR DESCRIPTION
**Pull request by wm75**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

**Comment by wm75** [comment]
@bernt-matthias not a huge expert on pipefail :) Am I doing things right here?

@davebx @astrovsky01 does the new background test data look reasonable to you? Previously that test failed silently.

@mvdbeek The proposed multiprocessing handling looks about right?

File: tools/bioext/bealign.xml [hunk 1]
```diff
@@ -18,7 +19,10 @@
     ## and replace any unwanted characters in the sequence names with underscores.
     ## This should not affect the actual alignment, since any non-standard character
     ## in the sequences is already ignored, but the possibility remains.
+    set -o pipefail;
     cat '$input' @SANITIZE@ reads.fa &&
+    ## bealign's parallel job executor uses the following env variable
+    export NCPU=\${GALAXY_SLOTS:-2} &&
```
Comment (mvdbeek) [inline tools/bioext/bealign.xml:25]: I would just prefix bealign with the variable and not export it. `NCPU=\${GALAXY_SLOTS:-2} bealign ...`. I guess in practice it doesn't make a difference, but NCPU is often but not always set by the DRM.

File: tools/bioext/bealign.xml [hunk 2]
```diff
@@ -18,7 +19,10 @@
     ## and replace any unwanted characters in the sequence names with underscores.
     ## This should not affect the actual alignment, since any non-standard character
     ## in the sequences is already ignored, but the possibility remains.
+    set -o pipefail;
```
Comment (bernt-matthias) [inline tools/bioext/bealign.xml:22]: Since there seems to be no pipe I guess this is not needed.

Comment (wm75) [reply tools/bioext/bealign.xml:22]: The pipe is hidden inisde the SANITIZE macro.

**Comment by bernt-matthias** [comment]
Looks good to me. Do we want/need to bump the version?

**Comment by bgruening** [comment]
suffix is bumped, imho. I think @mvdbeek comments need to be addressed and then we can go.

**Review by bgruening — APPROVED** [review]

**Review by bernt-matthias — APPROVED** [review]

**Comment by bernt-matthias** [comment]
@wm75 PR is fine as it is. Simplifications are welcome in a separate PR, anytime.

**Pull request merged**